Reference/Events/Understanding-Bootmanagers.md

```diff
@@ -0,0 +1,23 @@
+# Understanding Umbraco the BootManager
+
+After IIS started the W3 proces, umbraco will be launching.  There is a bootstrapper for the `Umbraco Application` which initializes all objects.
+
+The responsible objects for the startup are the [CoreBootManager](../../apidocs/csharp/api/Umbraco.Core.CoreBootManager.html) and [WebBootManager](/apidocs/csharp/api/Umbraco.Web.WebBootManager.html) where the latter includes the Web portion of the application.
+
+The boot managers initialize the [UmbracoApplication](apidocs/csharp/api/Umbraco.Web.UmbracoApplication.html) (the global.asax) object.  After it has initialized the UmbracoApplication, it will initialize the ApplicationContext.  
+
+The bootmanager will initialize the ApplicationContext with a e.g. the database Context, services Context, profiling and logger. It will also register the Application Startup handlers which will execute later using the [ApplicationEventsResolver](https://our.umbraco.org/apidocs/csharp/api/Umbraco.Core.ObjectResolution.ApplicationEventsResolver.html).
```

Comment:
with a e.g. ???

will initialize the ApplicationContext with: the database Context, services context...

Reply:
👍